pwndbg/commands/heap.py

```diff
@@ -22,6 +22,15 @@ def value_from_type(type_name, addr):
     gdb_type = pwndbg.typeinfo.load(type_name)
     return gdb.Value(addr).cast(gdb_type.pointer()).dereference()
```

Comment:
Might be good to move this to `pwndbg/typeinfo.py` and rename to `as_gdbtype(typename, addr)` or `cast_to(typename, addr)`.
@zachriggle What do you think?